src/NzbDrone.Core/Download/Clients/DownloadStation/TorrentDownloadStation.cs

```diff
@@ -188,7 +188,7 @@ protected override void Test(List<ValidationFailure> failures)
         {
             failures.AddIfNotNull(TestConnection());
             if (failures.Any()) return;
-            failures.AddIfNotNull(TestOutputPath());
+            failures.AddIfNotNull(TestOutputPath());            
```

Comment:
If you have the editorconfig extension installed, empty whitespace at the end of the line should be removed automatically.